packages/webdriverio/src/utils/getElementObject.ts

```diff
@@ -13,6 +13,26 @@ interface GetElementProps {
     isShadowElement?: boolean
 }
 
+interface ChromedriverErrorResponse {
```

Comment:
I think we should call this `DriverError` since all drivers have this error format according to the [spec](https://w3c.github.io/webdriver/#errors).